modules/utxo-lib/src/bitgo/wallet/psbt/PsbtOutputs.ts

```diff
@@ -79,6 +79,26 @@ export function findInternalOutputIndices(psbt: UtxoPsbt): number[] {
   return findWalletOutputIndices(psbt, rootNodes);
 }
 
+export function getChainAndIndexFromInternalOutput(output: PsbtOutput): { chain: number; index: number } | undefined {
```

Comment:
it's unclear how we are asserting that the output is _internal_ here - as I understand even external outputs can have derivation info.

Can we pass in the wallet xpubs as an argument maybe?